alignak/objects/config.py

```diff
@@ -1099,13 +1075,14 @@ def read_config_buf(self, buf):  # pylint: disable=R0912
 
             line_nb += 1
             # Remove comments
-            line = split_semicolon(line)[0].strip()
+            if not continuation_line:
```

Comment:
Because need get spaces and all info on multiline command for exemple when have html code, see https://github.com/Alignak-monitoring/alignak/blob/develop%2Benhanceunittests/test/test_command_multilines.py